### PR TITLE
chore: Splits the CHANGELOG into multiple files

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Splits the `CHANGELOG.md` into multiple files to better organize the contents of each release and also to deal with GitHub size limitation when displaying the file. Now every release will have its own file at `CHANGELOG/<version>.md`

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by john-bodley** [comment]
Thanks @michael-s-molina for the cleanup. My only comment is whether there's merit in keeping the `CHANGELOG.md` which contains a list of links to the version specific change logs.

**Comment by michael-s-molina** [comment]
> Thanks @michael-s-molina for the cleanup. My only comment is whether there's merit in keeping the CHANGELOG.md which contains a list of links to the version specific change logs.

@john-bodley I applied your suggestion.

**Review by john-bodley — APPROVED** [review]

File: RELEASING/email_templates/result_pmc.j2 [hunk 1]
```diff
@@ -20,15 +20,15 @@ To: {{ receiver_email }}
...
+Thanks to everyone who participated. The vote to release
 Apache {{ project_name }} version {{ version }} based on {{ version_rc }} is now closed.
 
 {% if vote_negatives|length > 0 -%}
 The vote did NOT PASS with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding +1, and {{vote_negatives|length}} -1 votes:
 {% elif vote_bindings|length > 2 -%}
 The vote PASSED with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding +1, and {{vote_negatives|length}} -1 votes:
 {% else -%}
-The vote is non conclusive with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding -1, and {{vote_negatives|length}} -1 votes:
+The vote is non conclusive with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding +1, and {{vote_negatives|length}} -1 votes:
```
Comment (rusackas) [inline RELEASING/email_templates/result_pmc.j2:31]: Nice catches. I just wonder if vote_nonbinding is either + OR -? I'm not seeing where this is tabulated to be sure. I'm guessing you're right though :D

**Review by rusackas — APPROVED** [review]
LGTM!!

**Review by dpgaspar — APPROVED** [review]
Nice!

File: RELEASING/email_templates/result_pmc.j2 (hunk 1 above)
Comment (michael-s-molina) [reply RELEASING/email_templates/result_pmc.j2:31]: It's defined [here](https://github.com/apache/superset/blob/8657fa753bbc20b6ff4159424d7385707156a349/RELEASING/generate_email.py#L104).

> A List of people with +1 non binding vote (ex: Ville)

**Pull request merged**